sunpy/instr/goes.py

```diff
@@ -106,6 +105,10 @@ def get_goes_event_list(timerange, goes_class_filter=None):
         e.g. 'M1', 'X2'.
 
     """
+
+    #import hek here as this is the only function using it
```

Comment:
This should be clearer, the reason this is here is not since its the only function using it per say, but to avoid importing and calling code that requires an optional dependency.

Reply:
Should I change and push again?

Reply:
Yes please.